src/app/pages/ReceiveRBTCPage/index.tsx

```diff
@@ -53,21 +66,56 @@ export const ReceiveRBTCPage: React.FC = () => {
             </div>
 
             <div className="tw-text-center">
-              <Link to="/fast-btc/transak">
-                <SelectBox>
-                  <div className="tw-flex">
-                    <div className="tw-z-20 tw-border tw-border-gray-5 tw-bg-gray-4 tw-flex tw-items-center tw-justify-center tw-h-12 tw-w-12 tw-rounded-full">
-                      <img src={dollarIcon} alt="dollar" />
-                    </div>
-                    <div className="tw-z-10 tw-border tw--ml-5 tw-border-gray-5 tw-bg-gray-4 tw-flex tw-items-center tw-justify-center tw-h-12 tw-w-12 tw-rounded-full">
-                      <img src={poundIcon} alt="pound" />
-                    </div>
-                    <div className="tw-border tw--ml-5 tw-border-gray-5 tw-bg-gray-4 tw-flex tw-items-center tw-justify-center tw-h-12 tw-w-12 tw-rounded-full">
-                      <img src={eruoIcon} alt="euro" />
+              <Tooltip
+                position="top"
+                hoverOpenDelay={0}
+                hoverCloseDelay={0}
+                interactionKind="hover"
+                content={
+                  <>
+                    {fastBtcReceiveLocked ? (
+                      t(translations.maintenance.fastBTCPortfolio)
```

Comment:
There are no links in footer, use `maintenance.fastBtc` error message with Discord link.

<img width="1925" alt="image" src="https://user-images.githubusercontent.com/14878713/226555554-a019af3c-6417-4e59-ad9e-602217c94db4.png">